.github/pull_request_template.md

```diff
@@ -5,7 +5,7 @@
 
 Closes # 
 
-### **Accomplishments:**
+### **Accomplishments:** (requirements)
```

Comment:
uncommit this change since it's not part of your issue, you can use this command which will revert the file back to the beginning of the branch (I think)

`git reset HEAD <file>`